File: Reverse.cs
using System;

class MainClass {
  public static string FirstReverse(string str) { 
  
    // code goes here  
    /* Note: In C# the return type of a function and the 
       parameter types being passed are defined, so this return 
       call must match the return type of the function.
       You are free to modify the return type. */
    
    string sol = "";
    foreach(char c in str){
        sol = c + sol;
        
    }
    
    return sol;
            
  }

  static void Main() {  
    // keep this function call here
    Console.WriteLine(FirstReverse(Console.ReadLine()));
  } 
   
}

File: Reverse.cpp
#include <iostream>
using namespace std;

string FirstReverse(string str) { 

  // code goes here   
  for(int i=0,j=str.length()-1;i<str.length()/2;i++,j--)
  {
    str[i]^=str[j]^=str[i]^=str[j];
  }
  return str; 
            
}

int main() { 
  
  // keep this function call here
  cout << FirstReverse(gets(stdin));
  return 0;
    
}